jenkins/job-configs/kubernetes-jenkins/kubernetes-e2e-kops-aws.yaml

```diff
@@ -22,9 +22,11 @@
         - shell: |
             # Fake provider to trick e2e-runner.sh
             export KUBERNETES_PROVIDER="kops-aws"
-            export AWS_CONFIG_FILE="${{WORKSPACE}}/.aws/credentials"
+            export AWS_CONFIG_FILE="/workspace/.aws/credentials"
```

Comment:
seems like we should maybe just move setting these vars into e2e-runner.sh to fix these bugs once and for all.